packages/netlify-cms-core/src/actions/config.js

```diff
@@ -111,12 +106,15 @@ export function loadConfig() {
     try {
       const preloadedConfig = getState().config;
```

Comment:
I don't believe this has a role any longer with the changes in this PR.

Reply:
Surely, I confess that I can't say if it is still relevant. Do you confirm that we can remove this ?

Reply:
Yeah, passing `bootstrapConfig` into `loadConfig` supersedes the old `preloadedConfig` behavior, you can drop it.